spyder/plugins/variableexplorer/widgets/namespacebrowser.py

```diff
@@ -284,6 +289,19 @@ def get_view_settings(self):
             settings[name] = getattr(self, name)
         return settings
 
+    def show_finder(self, state=None):

```

Comment:
`state=None` is not so clear here. Let's change that to `set_visible=False`, which I think better describes the role of this kwarg.